src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/ContestedCourtHelper.java

```diff
@@ -0,0 +1,287 @@
+            return getNorthWestFRC(caseData);
+        }
+        if (NORTHEAST.equalsIgnoreCase(region)) {
+            return getNorthEastFRC(caseData);
+        }
+        if (SOUTHEAST.equalsIgnoreCase(region)) {
+            return getSouthEastFRC(caseData);
+        }
+        if (WALES.equalsIgnoreCase(region)) {
+            return getWalesFRC(caseData);
+        }
+        return EMPTY;
+    }
+
+    private static String getWalesFRC(Map<String, Object> caseData) {
+        String walesList = (String) caseData.get(WALES_FRC_LIST);
+        if (NEWPORT.equalsIgnoreCase(walesList)) {
+            return getNewportCourt(caseData);
+        } else if (SWANSEA.equalsIgnoreCase(walesList)) {
+            return getSwanseaCourt(caseData);
+        }
+        return EMPTY;
+    }
+
+    private static String getSouthEastFRC(Map<String, Object> caseData) {
+        String southEastList = (String) caseData.get(SOUTHEAST_FRC_LIST);
+        if (KENT.equalsIgnoreCase(southEastList)) {
+            return getKentCourt(caseData);
+        }
+        return EMPTY;
+    }
+
+    private static String getNorthEastFRC(Map<String, Object> caseData) {
+        String northEastList = (String) caseData.get(NORTHEAST_FRC_LIST);
+        if (CLEAVELAND.equalsIgnoreCase(northEastList)) {
+            return getCleavelandCourt(caseData);
+        } else if (NWYORKSHIRE.equalsIgnoreCase(northEastList)) {
+            return getNwYorkshireCourt(caseData);
+        } else if (HSYORKSHIRE.equalsIgnoreCase(northEastList)) {
+            return getHumberCourt(caseData);
+        }
+        return EMPTY;
+    }
+
+    private static String getNorthWestFRC(Map<String, Object> caseData) {
+        String northWestList = (String) caseData.get(NORTHWEST_FRC_LIST);
+        if (LIVERPOOL.equalsIgnoreCase(northWestList)) {
+            return getLiverpoolCourt(caseData);
+        } else if (MANCHESTER.equalsIgnoreCase(northWestList)) {
+            return getManchesterCourt(caseData);
+        }
+        return EMPTY;
+    }
+
+    private static String getLondonFRC(Map<String, Object> caseData) {
+        String londonList = (String) caseData.get(LONDON_FRC_LIST);
+        if (CFC.equalsIgnoreCase(londonList)) {
+            return getLondonCourt(caseData);
+        }
+        return EMPTY;
+    }
+
+    private static String getMidlandFRC(Map<String, Object> caseData) {
+        String midlandsList = (String) caseData.get(MIDLANDS_FRC_LIST);
+        if (NOTTINGHAM.equalsIgnoreCase(midlandsList)) {
+            return getNottinghamCourt(caseData);
+        } else if (BIRMINGHAM.equalsIgnoreCase(midlandsList)) {
+            return getBirminghamCourt(caseData);
+        }
+        return EMPTY;
+    }
+
+    public static String getNottinghamCourt(Map<String, Object> caseData) {
+        return nottinghamMap.getOrDefault(caseData.get(NOTTINGHAM_COURTLIST), "");
+    }
+
+    private static Map<String, String> nottinghamMap = ImmutableMap.<String, String>builder()
```

Comment:
there's a solution in this repo that uses a JSON file to retrieve court details, it would be nice to re-use that to keep the solution consistent